src/System.Net.HttpListener/tests/HttpResponseStreamTests.cs

```diff
@@ -418,7 +419,7 @@ public void Dispose()
                 HttpListenerContext context = await listener.GetContextAsync();
 
                 // Disconnect the Socket from the HttpListener.
-                client.Dispose();
+                client.Close();
```

Comment:
FYI, the only difference between Close and Dispose is that Close includes a few additional traces in addition to calling Dispose:
https://source.dot.net/#System.Net.Sockets/System/Net/Sockets/Socket.cs,e16e31549d173065